### PR TITLE
Fixed some doc issues in reCAPTCHA

### PR DESCRIPTION
**Pull request by hwupathum**
Fixed some url issues in reCAPTCHA pages, and added global configuration. Merged recaptcha into the same page as the workflow

### CODE REVIEW

File: en/docs/guides/password-mgt/recover-password.md [hunk 1]
```diff
@@ -1,64 +1,85 @@
...
+    priority= "97"
+    enable = true
+    ```
 
-**Configure the email adapter to send emails**
+-   [Enable the email sending configurations]({{base_path}}/deploy/configure-email-sending) of the WSO2 Identity Server.
 
-Enable the email sending configurations of the WSO2 Identity Server as explained here.
+    !!! tip
+        Typically, the **AccountConfirmation** template is used to send email notifications.
```
Comment (himeshsiriwardana) [inline en/docs/guides/password-mgt/recover-password.md:27]:
```suggestion
        The **AccountConfirmation** template is used to send email notifications.
```

File: en/docs/guides/password-mgt/recover-password.md [hunk 2]
```diff
@@ -1,64 +1,85 @@
...
+    ```toml
+    [identity_mgt.password_reset_email]
+    enable_password_reset_email=true      
+    ```
+
+    !!! note
+        If you want to enable reCAPTCHA for password recovery via email, you can set `enable_recaptcha` true. See [Setting Up reCAPTCHA]({{base_path}}/deploy/configure-recaptcha) for more information.
+
+        ``` toml
+        enable_recaptcha=true
```
Comment (himeshsiriwardana) [inline en/docs/guides/password-mgt/recover-password.md:75]: Same suggestion as above.

File: en/docs/guides/identity-lifecycles/recover-username.md [hunk 3]
```diff
@@ -53,11 +55,57 @@ Follow the steps below to configure WSO2 Identity Server to enable username reco
...
+
+    ``` toml
+    [identity_mgt.username_recovery.email] 
+    enable_username_recovery= true
+    ```
+
+    !!! note
+        If you want to enable reCAPTCHA for username recovery, you can set `enable_recaptcha` true. See [Setting Up reCAPTCHA]({{base_path}}/deploy/configure-recaptcha) for more information.
+
+        ``` toml
```
[the remark this answers is not in the document]
Comment (hwupathum) [reply en/docs/guides/identity-lifecycles/recover-username.md:73]: `Should this be set under [identity_mgt.username_recovery.email] ` yes

File: en/docs/guides/identity-lifecycles/recover-username.md [hunk 4]
```diff
@@ -53,11 +55,57 @@ Follow the steps below to configure WSO2 Identity Server to enable username reco
 
 4.	Click **Update**. 
 
-### Use the REST API
+### Enable username recovery globally
+
+1. Open the `deployment.toml` file in the `IS_HOME/repository/conf` folder and add the following configuration:
+
+    !!! tip
+        To avoid any configuration issues, perform **Step-1** before starting the WSO2 Identity Server product instance.
```
[the remark this answers is not in the document]
Comment (hwupathum) [reply en/docs/guides/identity-lifecycles/recover-username.md:63]: yes

File: en/docs/guides/login/configure-recaptcha-for-single-sign-on.md [hunk 5]
```diff
@@ -61,6 +61,13 @@ You can either configure the reCAPTCHA for a tenant or configure it globally.
     enable_always=false
     max_attempts="3"
     ```
+
+!!! note
+    If you want to enable reCAPTCHA globally for all the workflows, the `forcefully_enabled_for_all_tenants` should be added as a property of `[recaptcha]` in the `deployment.toml` file.
+    ``` toml
+    forcefully_enabled_for_all_tenants=true
```
[the remark this answers is not in the document]
Comment (hwupathum) [reply en/docs/guides/login/configure-recaptcha-for-single-sign-on.md:68]: won't it be misunderstood by the readers?

**Review by himeshsiriwardana — APPROVED** [review]

File: en/docs/guides/login/configure-recaptcha-for-single-sign-on.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (himeshsiriwardana) [reply en/docs/guides/login/configure-recaptcha-for-single-sign-on.md:69]: this is the format that we have followed elsewhere. It's better to show where this property needs to reside.

**Pull request merged**